viofs/svc/virtiofs.cpp

```diff
@@ -136,10 +137,12 @@ struct VIRTFS
     // Maps NodeId to its Nlookup counter.
     std::map<UINT64, UINT64> LookupMap{};
 
-    VIRTFS(ULONG DebugFlags, bool CaseInsensitive, const std::wstring& MountPoint,
+    VIRTFS(ULONG DebugFlags, bool CaseInsensitive, 
```

Comment:
Trailing whitespace:
```suggestion
    VIRTFS(ULONG DebugFlags, bool CaseInsensitive,
```